Dockerfile

```diff
@@ -29,7 +31,11 @@ RUN BUILD_MONAI=1 FORCE_CUDA=1 python setup.py develop \
 
 # NGC Client
 WORKDIR /opt/tools
-RUN wget -q https://ngc.nvidia.com/downloads/ngccli_cat_linux.zip && \
+ARG NGC_CLI_SHA256_HASH="03c86402449469a6d12ac2fbbf5c8b460bdbed4f1e01f692e07eb1c144f6de0f"
```

Comment:
the file will be updated from time to time, please follow https://ngc.nvidia.com/setup/installers/cli for the check

Reply:
Sure, there will be changes. It is good practice however, to check the integrity of downloads. What may I do here? May I version the download so the download and SHA256 hash need to be changed accordingly? Or may I forfeit checking this download entirely?

Reply:
the link I pasted above has a command for md5 check

Reply:
Thanks. This looks good.